### ISSUE TITLE
Multiple filters, same target

### ISSUE THREAD

**Issue by lroggen** [chromawoods/instaFilta#9]
I'm trying to target target the same table rows with two different InstaFilters, one a checkbox and the other a text input field. It works well if I use one or the other but if I try to check a box and enter text, all of the td's in my table row disappear. 

Have you seen anything like this before? Can I even use two filters on the same targets?

        $('#task-search').instaFilta({
            scope: '.wrapper',
            targets: 'tr.task',
            markMatches: true
        });

        var instaFilta = $('#user-search').instaFilta({
            scope: '.wrapper',
            targets: 'tr.task'
        });

        var $userCheckboxes = $('.user-checkbox');
        $userCheckboxes.on('change', function() {
            var checkedCategories = [];
            $userCheckboxes.each(function() {
                if ($(this).prop('checked')) {
                    checkedCategories.push($(this).val());
                }
            });
            instaFilta.filterCategory(checkedCategories);
        });

Check Box:
![three](https://cloud.githubusercontent.com/assets/6265423/6933946/39741f7e-d7fb-11e4-9105-4ac5b12017e9.PNG)
Text Input and initial view:
![one](https://cloud.githubusercontent.com/assets/6265423/6933935/ec369d72-d7fa-11e4-824e-ae44f6da81e4.PNG)
Result of using both at once:
![two](https://cloud.githubusercontent.com/assets/6265423/6933934/ec346908-d7fa-11e4-9eee-abc0bbfb71ab.PNG)

**Comment by chromawoods** [comment]
Just to be clear of the expected result: So if you first check a checkbox, then start typing in the search field, the expected behavior is that the filtering process should only be applied to the remaining visible hits, right? So it should be like a filter-on-filter kind of thing? So any filtering process should ever only apply to whatever result is currently remaining.

I have thought about this feature before and I would like to implement it. I don't think you can do it currently. I'm going to have to look into it and get back with you.

**Comment by lroggen** [comment]
You are correct, the filter would limit everything to the checked item(s) and the search string. If it is not too complex, it would be a great feature!

**Comment by deutschehandarbeit** [comment]
I am really looking forward to this.

**Comment by brt-tito** [comment]
Yes, me too. That's a thing I really can use.

**Comment by chromawoods** [comment]
Right, beginning to feel the heat surrounding this issue. :fire:

I assure you all that I am considering this feature. Just not sure about the flow of things. I don't want to break any current functionality either. At the moment, the plugin kinda requires a text field as its main element, but it would be nice if this could be made more flexible.
